k8s/arroyo/Chart.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: v2
+name: arroyo
+description: Helm chart for the Arroyo stream processing engine
+
+type: application
+version: 0.1.0
+appVersion: "0.1.0"
+
+keywords:
+  - stream processing
+  - data
+  - real-time
+
+dependencies:
+  - name: prometheus
+    version: "^21"
+    repository: https://prometheus-community.github.io/helm-charts
+    condition: prometheus.deploy
+  - name: postgresql
+    version: "^12"
```

Comment:
Why not be on 15?

Reply:
This is version 12 of the helm chart, which is Postgres 15 (https://artifacthub.io/packages/helm/bitnami/postgresql)